converters/video_to_images/video_to_images.py

```diff
@@ -43,9 +48,14 @@ def convert(self, output_path=None, **kwargs):
             ffmpeg_command.run(overwrite_output=True)
             return frames_folder
         except ffmpeg.Error as e:
-            # Capture and show the error if it occurs
-            if e.stderr:
-                print(f"Error executing ffmpeg command: {e.stderr.decode('utf8')}")
-            else:
-                print("Unknown error while running ffmpeg command.")
-            raise
+            raise VideoConvertError(f"ffmpeg command execution failed: {e.stderr}", 500)
+
+
+    def validate_params(self, **kwargs):
+        validators = [ IntValidator(kwargs['fps'], True, "Fps"),
+                       ]
+        if kwargs['output_path']:
```

Comment:
When I send an invalid output_path, it doesn't return an error and ignores the output_path
![image](https://github.com/user-attachments/assets/649e94ad-a43f-418e-bc3f-cef969e11577)

Reply:
As I was checking I saw we left this parameter but we arent actually using it nor retreiving it in any part of the code so I removed the code referencing it coming from the endpoint. 
When making the request from postman the endpoint is currently only processing one parameter, the file with a fixed fps=1 
Fps is a valid parameter but only for the convert method itself